readme.md

```
@@ -132,6 +139,60 @@ node.get_info() -> Result<NodeInfo, ApiError> // returns NodeInfo and balances
 node.channel_info()
 ```
 
+#### LNURL & Lightning Address
+
+LNI supports paying to Lightning Addresses (`user@domain.com`) and LNURL endpoints. The `lnurl` module handles automatic resolution to BOLT11 invoices.
+
+**Rust**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)